### PR TITLE
Cleanup and triggers refactoring in SnazzyContacts Adapter

### PR DESCRIPTION
**Pull request by shterion**

### CODE REVIEW

**Comment by philecs** [comment]
Can you check in how far the README.md matches the [documentationGuidelines](https://github.com/openintegrationhub/Connectors/blob/master/Adapters/AdapterChecklists/AdapterDescriptionAndDocumentationChecklist.md)?

**Comment by shterion** [comment]
Readme is updated. The actions which are not used were already removed.

**Comment by shterion** [comment]
@philecs I just made the desired changes and fix some jsonata expressions. If you have noticed something missing just let me know.

File: Adapters/Examples/SnazzyContactsAdapter/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# elasticio-snazzycontacts-component
+# snazzycontacts-adapter
 > [Snazzy Contacts](https://snazzycontacts.com) Node.js component for [elastic.io platform](http://www.elastic.io "elastic.io platform")
```
Comment (philecs) [inline Adapters/Examples/SnazzyContactsAdapter/README.md]: Pls remove this one :)

**Pull request merged**